mrtt-ui/package-lock.json

```diff
@@ -30717,6 +30717,14 @@
         "websocket-driver": "^0.7.4"
       }
     },
+    "node_modules/sockjs/node_modules/uuid": {
```

Comment:
oof did not mean to commit this - was testing uuid before but don't need it

Reply:
I used npm uninstall but its still appearing here ... looks like a [known issue with npm ](https://stackoverflow.com/questions/64039455/package-lock-json-not-updated-after-removing-a-package-from-package-json)